### PR TITLE
implemented unspecified case reply

### PR DESCRIPTION
**Pull request by QIYUANYUAN**

### CODE REVIEW

File: sample-spring-boot-kitchensink/src/main/java/com/example/bot/spring/database/GQDBEngine.java [hunk 1]
```diff
@@ -74,6 +74,8 @@ public String query(String userID,String Text,String TourID) throws Exception{
 			   Text.toLowerCase().contains("how much time") ||
 			   Text.toLowerCase().contains("duration") ) 
 			{
+
```
Comment (Jekyll-SONG) [inline sample-spring-boot-kitchensink/src/main/java/com/example/bot/spring/database/GQDBEngine.java]: why not just throw an exception

**Comment by Crysitna** [comment]
should we also check tourID in "public void update(String UserID,String TourID)"??

**Comment by QIYUANYUAN** [comment]
@Jekyll-SONG Since it might be the case where user just type the tour name wrongly.

**Comment by QIYUANYUAN** [comment]
@Crysitna Well, since currently general question about a specific tour is implemented here, although we might consider combining this part with recommendation in the future.

**Pull request merged**